raven/base.py

```diff
@@ -636,7 +637,10 @@ def capture(self, event_type, data=None, date=None, time_spent=None,
             **kwargs)
 
         # should this event be sampled?
-        if self._random.random() < self.sample_rate:
+        if not sample_rate:
```

Comment:
oh actually if sample_rate is always None in the logging handler, and thus None gets passed here, that'd be a problem